src/client/components/voyage/StartVoyage.js

```diff
@@ -0,0 +1,241 @@
+import React, { Component } from "react";
+import Title from "./Title";
+import DateTime from "./DateTime";
+import InputField from "./InputField";
+import Dropdown from "./Dropdown";
+import Checkbox from "./Checkbox";
+import { ENGINE_METHOD_NONE } from "constants";
+
+class StartVoyage extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      vessels: [],
+      ports: [], //holds the list of harbours {name, lat, depth & lat}
+      optimisationType: [], // optimisationType: {id, optimisation_type}
+      departure_position: {}, // departure: {lat & lon}
+      arrival_position: {}, // arrival: {lat & lon}
+      vessel_id: 0,
+      ETD: "",
+      ETA: "",
+      optimisation_type: {},
+      hire_rate: "",
+      forward_draft: "",
+      aft_draft: "",
+      lfso_cost: "",
+      hfo_cost: "",
+      departureDate: "",
+      departureTime: "",
+      arrivalDate: "",
+      arrivalTime: "",
+      isChecked: false
+    };
+  }
+
+  handleDateInputChange = e => {
+    const target = e.target;
+    const { name, value } = target;
+    const dateDescr = name === "ETD" ? "departureDate" : "arrivalDate";
```

Comment:
Rename the variable dataDescr

Reply:
Comment resolved, variable description changed.